sim/rogue/rogue.go

```diff
@@ -84,6 +87,8 @@ type Rogue struct {
 	occultPoisonTick *core.Spell
 
 	instantPoisonProcChanceBonus float64
+	additivePoisonBonus 		 float64
```

Comment:
🔨 just a knit but could this be `additivePoisonBonusChance` so it's clearer that it affects the proc chance?

Reply:
DONE :)